ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusEditMessage.qml

```diff
@@ -59,7 +59,15 @@ Item {
                 size: StatusBaseButton.Size.Small
                 enabled: !!chatInputLoader.item && chatInputLoader.item.messageText.trim().length > 0
                 onClicked: {
-                    editCompleted(!chatInputLoader.item ? "" : chatInputLoader.item.messageText)
+                    let text = ""
+                    if (chatInputLoader.item) {
```

Comment:
If we updated the `sourceComponent` above to a StatusChatInput as the TODO says, we could get rid of that `if`. I did this to be sure to have backwards compatibility

Reply:
We can't really (directly) do this; a StatusQ component can not depend on something from the app

Reply:
> Can you do it in one if? 

I don't think so. I want to make sure to be backwards compatible, so I check if the item has a function called `getTextWithPublicKeys` before calling it, that way I am sure it will not throw.